cmd/capibmadm/utils/utils.go

```diff
@@ -50,9 +76,9 @@ func GetAccountID(ctx context.Context, auth *core.IamAuthenticator) (string, err
 	return *apiKey.AccountID, nil
 }
 
-// GetResourceGroupID returns ID of given resource group name.
+// GetResourceGroupID Function to get the resource group id.
```

Comment:
I think you can revert all the changes done in utils.go, you might have referred an old version of this code,  we have refactored and moved the funcs to client pkg.